Modules/Core/Common/include/itkVariableLengthVector.h

```diff
@@ -1152,7 +1152,7 @@ struct CanBeMultiplied
             mpl::And<mpl::IsNumber<TExpr1>, mpl::IsArray<TExpr2>>>
 {};
 
-/** Tells whether objects from two types can be multiplied.
+/** Tells whether objects from two types can be divided.
```

Comment:
The text "divided" does not match the name of the struct `CanBeMultiplied`.  Also, there is no description for this pull request.

Reply:
This is a docstring for the class which follows it.

I thought that a one-line PR does not need a description.